docs/cli.rst

```diff
@@ -343,6 +391,15 @@ Tools installation command
 You should use the :ref:`List Tool command` to display what tools and tool variant names are available.
     
 
+Legacy subcommands
+---------------
```

Comment:
Number of hyphen(-) is less than upper line

Reply:
Oops, not sure how that happened. Good catch!

Reply:
How do you feel about adding the `-W` flag to the `sphinx-build` commands in `tox.ini`? This would cause the build to fail if there are any warnings in the documentation. This would prevent silly mistakes like this in the future.

tox.ini:
```diff
commands =
-     sphinx-build {posargs:-E} -b html docs build/docs
-     sphinx-build -b linkcheck docs build/docs
+     sphinx-build {posargs:-E} -W -b html docs build/docs
+     sphinx-build -W -b linkcheck docs build/docs
```